cmd/pinniped-proxy/src/main.rs

```diff
@@ -17,17 +18,26 @@ async fn main() -> Result<()> {
     pretty_env_logger::init();
     let opt = cli::Options::from_args();
 
+    // Load the default certificate authority data on startup once.
+    let default_ca_data = fs::read_to_string(opt.default_ca_cert.clone())
+        .with_context(|| format!("unable to load default-ca-cert at {}", opt.default_ca_cert))?;
```

Comment:
`.with_context` is a trait of the `anyhow` package which enables adding context to a result before potentially returning an error. So instead of seeing just the following on the cmd line:

```
No such file or directory (os error 2)
```

you instead get:

```
Error: unable to load default-ca-cert at /var/run/secrets/kubernetes.io/serviceaccount/ca.crt

Caused by:
    No such file or directory (os error 2)
```

Reply:
Interesting! Much more meaningful errors. Having a look at the docs, it seems that for enabling backtraces, you have to set the envars `RUST_BACKTRACE=1` and/or `RUST_LIB_BACKTRACE=0`

Reply:
Yep, both the log-level and whether you display full back traces are just env vars.